lua/entities/ent_jack_sleepingbag.lua

```diff
@@ -0,0 +1,86 @@
+ENT.Type 			= "anim"
+ENT.PrintName		= "Sleeping Bag"
+ENT.Author			= "Basipek"
+ENT.Category			= "JMod - Misc"
+ENT.Spawnable			= true
+ENT.AdminSpawnable		= true
+
+
+if (CLIENT) then
+	function ENT:Draw()
+		self.Entity:DrawModel()
+	end
+elseif (SERVER) then
+
+	function ENT:Initialize()
+		self.Entity:SetModel("models/props_equipment/sleeping_bag1.mdl")
+		self.Entity:PhysicsInit(SOLID_VPHYSICS)
+		self.Entity:SetMoveType(MOVETYPE_VPHYSICS)
+		self:SetSolid( SOLID_VPHYSICS )
+		self.jackSleepingBagOwner=nil
```

Comment:
1) in all cases where we assign the owner, instead use `JMod.SetEZowner(ply)`
2) in all cases where we check the owner, instead check for `.EZowner`

this goes for the rest of the PR. `EZowner` is the standard we use in JMod